src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Spinner.Edit.cshtml

```diff
@@ -18,16 +21,20 @@
             <div class="input-group mb-2">
                 @if (settings.Minimum.HasValue)
                 {
-                <div class="input-group-prepend">
-                    <div class="input-group-text">@min</div>
-                </div>
+                    <div class="input-group-prepend">
+                        <div class="input-group-text">@min</div>
+                    </div>
                 }
-                <input asp-for="Value" typeof="text" class="form-control content-preview-select" type="number" required="@settings.Required" />
-                @if (settings.Maximum.HasValue)
-                {
+                <input asp-for="Value" typeof="text" class="form-control content-preview-select" placeholder="@settings.Placeholder" required="@settings.Required" />
                 <div class="input-group-append">
-                    <div class="input-group-text">@max</div>
+                    <button class="btn btn-outline-secondary" type="button" onclick="numericFieldSpinner('@(id)', @(scale), -@(step), @(min), @(max))"><i class="fa fa-minus" aria-hidden="true"></i></button>
```

Comment:
Could you try with btn-outline-light and show me a screenshot without commiting?

Reply:
Sure:

![image](https://user-images.githubusercontent.com/3008547/97480625-f6e33f00-1953-11eb-8d82-40f95bbf70ef.png)

Reply:
They need to add one that matches the form input border color. 😢

Reply:
Agreed. `btn-outline-secondary` was visually the best (probably why it's used on https://getbootstrap.com/docs/4.0/components/input-group/#button-addons as a demo).